wiki/Tournaments/DACH4K/2023/en.md

```diff
@@ -0,0 +1,382 @@
+4. Reschedules will only be accepted if both players **and** staff team agreement , posted in the Discord server before Friday 0:00 (UTC+2). If reschedule posted after this time , it won't be accepted.
+5. Repeated toxic behaviour or cheating will be banned from the tournament.
+
+### Registration
+
+1. Only players who have the following national flags , or others things could prove they lived in following countries would allowed to play in this tournament.
+   - Germany
+   - Austria
+   - Switzerland
+2. Players's mania 4k rank should between 1000 ~ 40000 at registration phase.
+3. every player should join the Discord server , this makes it easier to reach the player and ensures that he/she gets all current information.
+4. All staff members except streamers, commentators and graphic designers are not allowed to participate in this tournament as a player.
+
+### Match procedure
+
+#### Qualifier
+
+1. Each player may sign up into qualifier lobby in advace.
+   - if none of default lobby suit for players, players can request an custom lobby before Monday 0:00 (UTC+2) on qualifier week. 
+2. 15 minutes before the qualifier starts, players will be made aware of the qualifier via the ref. 5 minutes before the actual start, players will be invited to the lobby.
+3. In the lobby, all qualifier maps will be played 2 times in a row, with an optional 5 minute break between each run.
+4. Maps in qualifiers stage will be:
+   - Stage 1: SV
+   - Stage 2: RC Acc
+   - Stage 3: LN
+   - Stage 4: HB
+   - Stage 5: Mixed RC
+
+#### KO stage
+
+1. 15 minutes before the match starts, players are made aware of the match via the ref and asked for their warm-ups.
+2. 5 minutes before the actual start, players are invited to the lobby. Once both players are in the lobby, the warm-ups will be played.
+   - The maximum drain time of warm-ups is 3 minutes.
+3. After the warm-ups , both players enter the `!roll` command. Player with the higher number may decide whether to protect a map first, banned one second and pick one first or vice versa.
+   - Protect : This map may no longer be banned for this match.
+   - Ban : This map may no longer be picked for this match.
+   - Pick : A player says a map that is in the current map pool, which is then played.
+4. After protects and bans are determined, each player has 2 minutes to pick a map, if this time is exceeded, a map that is still left is picked randomly. When both players are ready, the map is started with a 10 second timer.
+5. Each player has the option of a 2 minute time-out during each match.
+6. If a player disconnects in the first 30 seconds of the map, the map may be repeated, this only allowed once for each player per match. If a player is disconnected after 30 seconds or it happend twice , Player will lose this map.
+7. A player wins the match if the required number of points is reached, if both players are one point away from winning the match the tiebreaker is played, otherwise it may not be picked.
```

Comment:
```suggestion
7. A player wins the match if the required number of points is reached. If both players are one point away from winning the match, the tiebreaker is played, otherwise it may not be picked.
```